app/src/main/res/layout/prompt_choice.xml

```diff
@@ -56,7 +56,6 @@
         android:layout_marginEnd="20dp"
         android:divider="@android:color/transparent"
         android:smoothScrollbar="true"
-        android:overScrollMode="never"
```

Comment:
I guess the question here is why. There is no explanation why we are doing that.

Reply:
`overScrollMode` is responsible for the "rubber band" effect in the lists. Here is a comparison on Pico (the Meta effect is slightly different).

- current:

https://github.com/user-attachments/assets/09262dcf-dc83-4883-9308-dbd12ab391c2

- this PR:

https://github.com/user-attachments/assets/c73822af-8a6c-4c50-b82f-f8128462d5e6

Reply:
yeah I know what the rubber band effect is, the question is why do we want to remove it, and secondly what's the relationship with adding search in the library?

Reply:
It introduces an unnatural effect when scrolling long lists that doesn't look good IMHO. On Oculus, it slows down the scroll at the edges, and compresses and expands the items.

https://github.com/user-attachments/assets/38d0e1f3-116c-4523-8d5b-e32b8f88afa0

For this PR I spent a long time using these lists and looking at their implementation, so I noticed this issue.

This particular change is included as a separate commit.